Alarm/AlarmDetailTableViewController.swift

```diff
@@ -77,9 +77,6 @@ class AlarmDetailTableViewController: UITableViewController, AlarmScheduler {
         didSet {
             if isViewLoaded {
                 updateViews()
-            } else {
-                loadView()
```

Comment:
Thanks for catching this. It's never appropriate to call `loadView()` like this.